### PR TITLE
Improves tab menu

### LINKED ISSUE #4796: Add Ctrl+Shift+T hotkey to reopen last closed tab
**Is your feature request related to a problem?**
Oh no, I just closed my [very important content] tab, and I want to quickly reopen it.

**Describe the solution**
If there was a hotkey for this, I could just reopen the tab I just closed,

**Describe alternatives you've considered**
I could try to reopen the tab by navigating to the corresponding folder, but what if it's very deep in the structure?

**Additional comment**
How many tabs do we want to keep in the history? Should I be able to reopen a tab that was closed, five million tabs ago?

Also @yaichenbaum :)

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Improves tab menu in horizontal mode. 
Closes #4796

**Details of Changes**
Add command to reopen last closed.
show shortcuts in the menu.

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
tabmenu

### CODE REVIEW

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 1]
```diff
@@ -23,10 +23,13 @@
                         <MenuFlyoutItem
                             x:Uid="HorizontalMultitaskingControlNewTab"
                             Click="{x:Bind vm:MainPageViewModel.AddNewTabAtIndex}"
-                            Text="New tab (Ctrl+T)">
+                            Text="New tab">
                             <MenuFlyoutItem.Icon>
                                 <FontIcon FontFamily="{StaticResource CustomGlyph}" Glyph="&#xF113;" />
                             </MenuFlyoutItem.Icon>
+                            <MenuFlyoutItem.KeyboardAccelerators>
+                                <KeyboardAccelerator Key="T" Modifiers="Control" />
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:31]:
```suggestion
                                <KeyboardAccelerator Key="T" Modifiers="Control" IsEnabled="False"  />
```

File: Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml [hunk 2]
```diff
@@ -51,6 +54,15 @@
                             Click="{x:Bind vm:MainPageViewModel.CloseTabsToTheRight}"
                             DataContextChanged="MenuItemCloseTabsToTheRight_DataContextChanged"
                             Text="Close tabs to the right" />
+                        <MenuFlyoutItem
+                            x:Name="MenuItemReopenClosedTab"
+                            x:Uid="HorizontalMultitaskingControlReopenClosedTab"
+                            Click="{x:Bind vm:MainPageViewModel.ReopenClosedTab}"
+                            Text="Reopen closed tab">
+                            <MenuFlyoutItem.KeyboardAccelerators>
+                                <KeyboardAccelerator Key="T" Modifiers="Control,Shift" />
```
Comment (yaichenbaum) [inline Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml:63]:
```suggestion
                                <KeyboardAccelerator Key="T" Modifiers="Control,Shift" IsEnabled="False" />
```

**Review by yair100 — CHANGES_REQUESTED** [review]

**Comment by cinqmilleans** [comment]
@yair100 You are right.

**Review by yair100 — APPROVED** [review]

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**